docs/topics/maven-arguments.adoc

```diff
@@ -134,7 +134,7 @@ The `targetTechnologies` argument now provides version support, which follows th
 ====
 When migrating to JBoss EAP, be sure to specify the version in the target, for example, `eap:6`. Specifying only `eap` will run rulesets for all versions of JBoss EAP, including those not relevant to your migration path.
 
-See link:{ProductDocGettingStartedGuideURL}#migration_paths[Supported migration paths] in the {ProductShortName} _{GettingStartedBookName}_ for which JBoss EAP version is appropriate for your source platform.
+See link:{ProductDocIntroToMTAGuideURL}#migration_paths[Supported migration paths] in the {ProductShortName} _{IntroToMTABookName}_ for which JBoss EAP version is appropriate for your source platform.
```

Comment:
Fluff. I saw this before but forgot to comment on it.

Change "for which JBoss EAP version is appropriate for your source platform" to "for the appropriate JBoss EAP version." I think this appears in other places.